rq_dashboard/templates/rq_dashboard/dashboard.js

```diff
@@ -6,11 +6,15 @@ var url_for = function(name, param) {
     else if (name == 'workers') { url += 'workers.json'; }
     else if (name == 'cancel_job') { url += 'job/' + encodeURIComponent(param) + '/cancel'; }
     else if (name == 'requeue_job') { url += 'job/' + encodeURIComponent(param) + '/requeue'; }
+    else if (name == 'requeue_finished_job') { url += 'job/' + encodeURIComponent(param) +  '/finished/requeue'; }
```

Comment:
Why would we want to requeue a finished job?

Reply:
Running that job again ?